src/codeql.ts

```diff
@@ -189,7 +189,10 @@ export async function getBqrsInfo(
       `Unable to run codeql bqrs info. Exit code: ${bqrsInfoOutput.exitCode}`
     );
   }
-  return deserialize(bqrsInfoOutput.stdout);
+  return validateObject(
```

Comment:
Will this message be confusing to users/us? The message might say something like "missing `compatibleQueryKinds`", while the actual property in the file could be called `compatible_query_kinds`. Would it make sense to remove the camelization in a future PR?